app/views/home/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<nav class="navbar navbar-default navbar-expand-lg fixed-top">
+  <div class="container flex justify-content-between align-items-center">
+    <span></span>
+    <h6 class="title">CATEGORIES</h6>
+    <div></div>
+  </div>
+</nav>
+
+<div class="d-flex justify-content-between align-items-center bg-white p-2">
+  <h6 class="pt-2">Categories</h6>
+  <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary' %>
+</div>
+
+<div class="mb-5">
+  <% if false %>
+    <div class="d-flex flex-column justify-content-center align-items-center m-5">
+      <p>Category is empty.</p>
+      <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary' %>
+    </div>
+  <% else %>
+    <% 1.times do %>
+      <%= link_to controller: 'groups', action: 'show', id: 1 do %>
+        <div class="d-flex justify-content-between align-items-center bg-white p-2 gap-6 m-2 shadow-sm">
+          <div class="d-flex justify-content-start align-items-center gap-3">
+            <img src="<%= 'group.icon' %>" height="100" width="100">
+            <div class="flex-column d-flex justify-content-between mt-4">
+              <h6 class="pb-3 text-black-60"><%= 'group.name.capitalize' %></h6>
+              <p class="text-black-50"><%= 'group.created_at.strftime("%Y-%m-%d")' %></p>
+            </div>
+          </div>
+          <div class="fw-bold">$100</div>
+        </div>
+      <% end %>
+    <% end %>
+  <% end %>
+</div>
+
+<div class="position-fixed bottom-0 w-100">
+  <div class="d-flex justify-content-center align-items-center bg-white p-2">
+    <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary w-full w-50' %>
+  </div>
+</div>
```

Comment:
- [ Required ] I think it would be nice if the design for your app matched that provided in the [Behance design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for this project as suggested by the previous reviewer. Currently, your design is different from the design provided in the following ways.
   - [ ] The fixed button at the bottom of the homepage does not have the `#5fb523` color it has on the Behance design In your design it's black.
    - [ ] The navigation bar on the Behance design has the color `#3778c2` whereas in your design it has a greenish color `#2eca6a` which is not part of the color scheme for this project.
    - [ ] The fixed button at the bottom of the page should have a width that fits the whole page, in your design it does not do so

Please see the screenshots below and rectify this.

## Screenshots
| Your design | Project design |
|-----------------|---------------------|
|![Screenshot from 2023-10-21 09-24-42](https://github.com/SAZZAD1Q2/BudgetMate/assets/16392046/02a902b4-236f-472e-a0f2-a1d7c243be66) | ![Screenshot from 2023-10-21 09-37-15](https://github.com/SAZZAD1Q2/BudgetMate/assets/16392046/73ce285c-cc08-40f4-8ded-953f6b1765ef) |